assets/src/edit-story/components/canvas/useCanvasSelectionCopyPaste.js

```diff
@@ -145,6 +158,26 @@ function useCanvasSelectionCopyPaste(container) {
             });
             evt.preventDefault();
           }
+          // If we're not copying a Story element, assume copying text.
+          if (!copyingStoryElement && copiedContent.trim().length) {
+            const props = {
```

Comment:
Pls switch to `useInsertElement` instead of `createNewElement + addElement`. And it will also automatically prefill some of the default values. And most importantly it will also calculate the correct text height.

Reply:
Interestingly, I was originally using `useInsertElement` which actually seemed to not pre-fill the default values and the correct text height neither -- switched to `createNewElement` + `addElement` due to that.

Will test again tomorrow.

Reply:
Okay, now I see why the height calculation wasn't working before -- I was mistakenly adding too many params to `useInsertElement`. Works great with the right amount of params.